index.js

```diff
@@ -73,16 +73,22 @@ try {
     core.info(`Task comment: "${TASK_COMMENT}"`);
     core.info(`Mark complete: "${MARK_COMPLETE}"`);
     core.info(`PR body: ${PULL_REQUEST.body}`);
-    let taskComment = null,
-        parseAsanaURL = null;
+    let taskComment = null
 
     if (!ASANA_PAT) {
         throw { message: "Asana PAT not found!" };
     }
     if (TASK_COMMENT) {
         taskComment = `${TASK_COMMENT} ${PULL_REQUEST.html_url}`;
     }
-    while ((parseAsanaURL = REGEX.exec(PULL_REQUEST.body)) !== null) {
+
+    const foundAsanaURLs = [...(PULL_REQUEST?.body?.matchAll(REGEX) || [])];
+
+    if (foundAsanaURLs.length === 0) {
+        core.warning("0 Asana URLs found matching the `trigger-phrase`.")
```

Comment:
question: What kind of warning does this trigger? If it's the kind that appears for deprecations (https://github.com/mbta/api/actions/runs/8918841034 for example) maybe it should be an info or debug statement instead?

Reply:
Oh it does seem to show up like that, I hadn't noticed.

https://github.com/mbta/skate/actions/runs/8917889101
https://github.com/mbta/skate/actions/runs/8917889101/job/24491646744
![image](https://github.com/mbta/github-asana-action/assets/6913198/70c8184c-417d-4325-8d00-03389a00e94b)

I'm on the fence, I'm okay with a changing this to info or debug (do you have a preference?) but I kinda like this warning.. I'm not sure what that would look like for other teams though.

Reply:
I'd vote for debug, since that's why you originally wanted to add it.

Reply:
Fixed https://github.com/mbta/github-asana-action/compare/ae03b57b31b52f3b54d55c98aedfd27bc2a5e816..e09f49cce047e05d0bd8e369283e35c5e4e9869a